lambda_code/elasticbeanstalk/Tags/index.py

```diff
@@ -0,0 +1,32 @@
+from cfn_custom_resource import CloudFormationCustomResource
+
+
+class Tags(CloudFormationCustomResource):
+    def validate(self):
+        self.environmentArn = self.resource_properties.get('EnvironmentArn')
```

Comment:
Since these properties are required, I'd access them using `self.resource_properties['EnvironmentArn']` instead of using `.get()`. `.get()` will return None if the requested key is not found; `[]`-access will raise an Exception in that case (and Fail the action on this resource)